### ISSUE TITLE
Bump up Lodash version

### ISSUE THREAD

**Issue by ak-08** [peterdemartini/html5-to-pdf#78]
The version of Lodash used creates a vulnerability which can be addressed by patching the current version to `4.17.16` 

https://app.snyk.io/vuln/SNYK-JS-LODASH-567746

**Comment by peterdemartini** [comment]
It's not locked down to that version specifically so it's not a major deal but feel free to create a PR and we can make it explicit.

**Comment by ak-08** [comment]
> It's not locked down to that version specifically so it's not a major deal but feel free to create a PR and we can make it explicit. 

Yeah in the repo I use, I've upgraded it to a non vulnerable version as a resolution so it's not too bad.

I'll create a PR for it though. 

P.S. you could look at something like renovate to maintain these dependencies for you :)
